sdk/metric/pipeline.go

```diff
@@ -104,13 +104,24 @@ func (p *pipeline) addCallback(callback func(context.Context)) {
 	p.callbacks = append(p.callbacks, callback)
 }
 
+// The key type is unexported to prevent collisions with context keys defined in
+// other packages.
```

Comment:
This comment describes the Go type system.

```suggestion
// callbackKey is a context key type used to identify context that came from the SDK.
```